src/app/attendance/page.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { PageLayout } from "../layout/PageLayout";
+import { AttendanceList } from "./components/AttendanceList";
+import { getAttendace } from "./actions/getAttendance";
+import { getMembers } from "../members/actions/getMembers";
+
+const Page = async () => {
+  const attendance = await getAttendace();
+  const members = await getMembers();
```

Comment:
Maybe we can remove `members` as a prop and instead we can just map over attendances and map users here. Wdyt?